install_config/cluster_metrics.adoc

```diff
@@ -359,42 +394,45 @@ HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com | oc create -f -
 ====
 The following command sets the Hawkular Metrics route to use
 `hawkular-metrics.example.com` and is deployed without persistent storage.
-Remember, this is being deployed without persistent storage, so metric data loss
+Remember, this is being deployed without persistent storage, so metrics data loss
 can occur.
 
-[options="nowrap"]
 ----
-$ oc process -f metrics.yaml -v \
-HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,USE_PERSISTENT_STORAGE=false \
-| oc create -f -
+$ oc process -f metrics-deployer.yaml -v \
+    HAWKULAR_METRICS_HOSTNAME=hawkular-metrics.example.com,USE_PERSISTENT_STORAGE=false \
+    | oc create -f -
 ----
 ====
 
-[[reencrypting-route]]
+[[metrics-reencrypting-route]]
 == Using a Re-encrypting Route
```

Comment:
I think all of this section (except maybe the example) can be one procedure instead of a bunch of little separate steps.